_example/webntp.ts

```diff
@@ -117,65 +87,19 @@ module WebNTP {
 
         // get_connection from the pool
         private get_connection(url : string): Connection {
-            if (this.pool.has(url)) {
-                // reuse connection
-                return this.pool.get(url);
+            let c = this.pool.get(url);
+            if (c !== undefined) {
+                return c;
             }
+
             // create new connection
-            const c = new Connection(url);
+            c = new Connection(url);
             this.pool.set(url, c);
             return c;
         }
 
-        get(url : string): Promise<Result> {
+        async get(url : string): Promise<Result> {
             return this.get_connection(url).get();
         }
-
-        get_multi(url: string, samples: number): Promise<Result> {
```

Comment:
drop get_multi feature temporary.
It is too difficult for me to implement it correctly...

I will try it later.